package.json

```diff
@@ -27,9 +27,11 @@
 		"start-server": "http-server &",
 		"test": "npm run start-server && backstop test"
 	},
+	"dependencies": {
+		"http-server": "^13"
```

Comment:
I am not sure that http-server is  a dependency.  There are other http servers that will work as well. 

And the code doesn't care if there is a http server running.  It is only if you want to demo this that this is useful.

Reply:
> I am not sure that http-server is a dependency. There are other http servers that will work as well.

Yes, not just others, but any other.

> And the code doesn't care if there is a http server running. It is only if you want to demo this that this is useful.

And that is the point: if we let it be a dependency, in README the process to see something working would be easier, would be the default: install dependencies and run a script. And if that install the development dependencies unnecessary things would be installed.

Reply:
Ok. I see your point. If this turns out not quite right in the future it can be changed. But, sure, let's go with this.